ocf_datapipes/training/example/nwp_pv.py

```diff
@@ -49,7 +49,10 @@ def nwp_pv_datapipe(
         .fork(2, buffer_size=BUFFER_SIZE)
     )
 
-    if configuration.input_data.nwp.nwp_provider == "UKMetOffice":
+    if (
+        configuration.input_data.nwp.nwp_provider == "UKMetOffice"
+        or configuration.input_data.nwp.nwp_provider == "ukv"
+    ):
```

Comment:
Could make this slightly tidier using:
`if  configuration.input_data.nwp.nwp_provider in [ "UKMetOffice", "ukv"]`?